example/disasm/dis_binary.py

```diff
@@ -0,0 +1,28 @@
+import sys
+from miasm2.analysis.binary import Container
+from miasm2.analysis.machine import Machine
+
+fdesc = open(sys.argv[1], 'rb')
+
+# The Container will provide a *bin_stream*, bytes source for the disasm engine
+# It will prodive a view from a PE or an ELF.
+cont = Container.from_stream(fdesc)
+
+# Instantiate an architecture depending on the binary
```

Comment:
"Instantiate an architecture" may be misleading. Based on what you wrote above, maybe something like:
"The Machine, instantiated with the detected architecture, will provide tools (disassembler, etc.) to work with this architecture"

Reply:
Done